libs/sdk-core/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ### Breaking change
 - `setField` method replaces `addField` and `updateField` which were identical.
 
+### Improvements
+- Manage properly 529 status (partial results) on chat endpoint
```

Comment:
The 529 error does not mean exactly we have partial results.

Here is the thing:
- The `/chat` endpoint made (internally) a call to `/find` and got a 206 status (which means "Partial results")
- As it makes no sense to generate an answer based on partial results, `/chat` returns a 529, which means "Service is overloaded" (we decided to use 529 here because we know that we get partial results when the system is overloaded)

So we should no mention "partial results" here (enough that's the original problem), the real error is "Service overloaded"

Reply:
oh, I'll rework it